pxr/base/tf/hash.h

```diff
@@ -94,6 +97,33 @@ TfHashAppend(HashState &h, std::vector<T> const &vec)
     h.AppendContiguous(vec.data(), vec.size());
 }
 
+// Support std::set
```

Comment:
tiniest nit: the other comments for the other "Support <type>." have '.' at the end. might as well keep this consistent.